token-swap/js/src/index.ts

```diff
@@ -461,9 +466,10 @@ export class TokenSwap {
     userTransferAuthority: Account,
     amountIn: number | Numberu64,
     minimumAmountOut: number | Numberu64,
+    confirmOptions: ConfirmOptions,
```

Comment:
nit: let's keep these consistent
```suggestion
    confirmOptions?: ConfirmOptions,
```